doc/new-book/routing.adoc

```diff
@@ -288,11 +307,13 @@ address {
 }
 ----
 
-`prefix`:: The address prefix that matches the broker queue to which you want to route messages.
+`prefix` | `pattern`:: The address prefix that matches the broker queue to which you want to route messages.
 +
-//tag::prefix-matching[]
-The prefix can match either an exact address or a segment within an address that is delimited by either a `.` or `/` character. For example, the prefix `my_address` would match the address `my_address` as well as `my_address.1` and `my_address/1`. However, it would not match `my_address1`.
-//end::prefix-matching[]
+The address prefix or pattern that matches the broker queue to which you want to send messages. You can specify a prefix to match an exact address or segment of an address. Alternatively, you can specify a pattern to match an address using wildcards.
```

Comment:
Line 312 essentially duplicates 310, or at least that was my impression on the read through.  Perhaps replace the description on 310 with the one in 312?